### DISCUSSION TITLE
Users can't delete pages although delete:pages is allowed in Group Permissions

### DISCUSSION THREAD

**Discussion by mittkkoo** [requarks/wiki#6674] [Help / Questions] [answered]
The users in a group are not able to delete pages. The garbage icon will not appear in the Edit page menu. Only the Admin is able to delete pages. Here are the settings under _Edit Group / Permissions_. `delete:pages` is allowed. 
![Screenshot_bienen](https://github.com/requarks/wiki/assets/76665232/9871a23d-8bc2-447f-b2b1-4390592d6a96)
I have no page rules, because I want that all users in this group can delete any page.

**Comment by P4mes4n** [comment, marked as answer]
You need page rules. Otherwise your permissions won't work.
https://docs.requarks.io/groups

> Do I need to define page rules if I want users to have the permissions applied everywhere?
> 
> Yes, for content permissions only. You need to define at least 1 page rule with the selected permissions and using the rule pattern where Path starts with... and a blank value.
